learners/setup.md

```diff
@@ -2,53 +2,164 @@
 title: Setup
 ---
 
-FIXME: Setup instructions live in this document. Please specify the tools and
-the data sets the Learner needs to have installed.
+## Download Data
+
+During these lessons we will be using two microscopy images and creating images
+of our own. Create an empty folder somewhere on your computer to use during the
+course, this will be your `working directory`. You can name it anything you
+like but we recommend `microscopy-ideas`.
+
+Download
+[00001_01.ome.tiff](
+https://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/MitoCheck/00001_01.ome.tiff)
+ ([CC BY 4.0 Licence](
+https://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/MitoCheck/readme.txt))
+ from the open microscopy environment pages to your working directory.
+
+Download
+[Plate1-Blue-A-12-Scene-3-P3-F2-03.czi](
+https://downloads.openmicroscopy.org/images/Zeiss-CZI/idr0011/Plate1-Blue-A_TS-Stinger/Plate1-Blue-A-12-Scene-3-P3-F2-03.czi)
+ ([CC BY 4.0 Licence](
+https://downloads.openmicroscopy.org/images/Zeiss-CZI/idr0011/readme.txt))
+from the open microscopy environment pages to your working directory.
 
-## Data Sets
+## Software Setup
 
-<!--
-FIXME: place any data you want learners to use in `episodes/data` and then use
-       a relative link ( [data zip file](data/lesson-data.zip) ) to provide a
-       link to it, replacing the example.com link.
--->
-Download the [data zip file](https://example.com/FIXME) and unzip it to your Desktop
+During these lessons we will use the [napari](https://napari.org/stable/)
+image viewer and the
+[napari-aicsimagio](https://github.com/AllenCellModeling/napari-aicsimageio)
+ plugin to load imaging data files of various file formats.
+For beginners we recommend using Anaconda to install and manage these
+packages. The first step is to
+[download and install Anaconda](https://www.anaconda.com/download#downloads).
 
-## Software Setup
+The remaining instructions are written assuming you will be installing and
+running the software from a terminal. To open a terminal follow the
+instructions for your operating system in the
+['Starting Conda' section of the Anaconda getting started pages](
+https://conda.io/projects/conda/en/latest/user-guide/getting-started.html#starting-conda).
```

Comment:
Conda updated their user guide a few weeks ago. Unfortunately, this removed references to the Anaconda prompt (for Windows), instead using cmd/powershell. There are extra steps required to get conda working with these terminals, vs anaconda prompt which is a lot easier.

Instead of linking to the anaconda getting started pages, could you include the instructions for opening anaconda prompt, or a terminal for windows/mac/linux directly in the setup instructions?

Reply:
I've added this, let me know what you think.

Reply:
Thanks! Could you change the Windows line to: Click Start > Search for Anaconda Prompt > Click to open

As I mentioned above, it's a lot easier to use conda on Windows with the Anaconda prompt rather than cmd/powershell